src/routes/templates/templates.json

```diff
@@ -670,5 +670,22 @@
 			"stores-and-state",
 			"svelte"
 		]
+	},
+	{
+		"title": "spatz2",
+		"url": "https://spatz2.engage-dev.com",
+		"repository": "https://github.com/engageintellect/spatz-2",
+		"description": "An opinionated template for building full-stack Svelte applications ridiculously fast. Includes auth, database, admin panel, user dashboard, themes, icons, and more.",
+		"npm": "svelte-lorem-ipsum",
+		"categories": [
+			"svelte",
+			"svelte-add",
+			"sveltekit",
+			"database",
+			"ssr",
+			"preprocessors",
+			"integrations",
+			"mobile"
```

Comment:
The maximum allowed number of categories is set to `6`

Reply:
done ✅